app/views/servers/check_installation.js.erb

```diff
@@ -2,5 +2,20 @@
   $("[data-server-status~=checking]").addClass("hidden");
   $("[data-server-status~=connected]").removeClass("hidden");
 <% else %>
+  <% if @server.install_step == 1 %>
+    $(".step1").addClass("info").find("td:first-child").html("<%= j icon "circle-o-notch fa-spin" %>");
+  <% elsif @server.install_step == 2 %>
+    $(".step1").removeClass("info").addClass("success").find("td:first-child").html("<%= j icon "check-square-o" %>");
+    $(".step2").addClass("info").find("td:first-child").html("<%= j icon "circle-o-notch fa-spin" %>");
+  <% elsif @server.install_step == 3 %>
+    $(".step1").removeClass("info").addClass("success").find("td:first-child").html("<%= j icon "check-square-o" %>");
```

Comment:
We *could* consider doing fancy things with css pseudo elemens. Since these are just font awesome icons you're changing.

So something like:
```css
.step {
   :before { content:  "\f046"; }
   &.processing { content:  "\f046"; }
   &.success { content:  "\f046"; }
}
```
But then with the proper content values of course.

In that case you just have to swap classes :)